presto-tpcds/src/main/java/com/facebook/presto/tpcds/TpcdsMetadata.java

```diff
@@ -236,7 +236,7 @@ public static double schemaNameToScaleFactor(String schemaName)
         }
 
         try {
-            return Integer.parseInt(schemaName.substring(2));
+            return Double.parseDouble(schemaName.substring(2));
```

Comment:
Do you rely on the fact "0.01" can be converted to double and back to string and, alas, it will yield "0.01" back?
Maybe `BigDecimal` (or `String`) would be a better choice?